### ISSUE TITLE
db.collection is not a function

### ISSUE THREAD

**Issue by MarcJamesIO** [stripe/stripe-firebase-extensions#469]
# Bug report

- Extension name: `firestore-stripe-payments`

## Describe the bug

I am following your tutorial here: https://firebase.google.com/codelabs/stripe-firebase-extensions#4

And getting the error db.collections is not a function. Is the tutorial completely out of date?

I am using the latest version of Firebase.

## To Reproduce

Just run through above tutorial with the latest version of Firebase.

## Expected behavior

The tutorial to not throw any errors.

**Comment by ajdrake** [comment]
Yes, I believe so. Firebase has upgraded the firestore sdk to version 9 modular, which uses a new approach to interacting with the datatabase. You can see the new approach described in their documentation to migrate from version 8 to 9 here.

https://firebase.google.com/docs/web/modular-upgrade#example_2_refactoring_a_function
